rocket/Rocket.cs

```diff
@@ -36,7 +36,7 @@ public double Distance
         public string Name { get; set; }
         public double Fuel;
```

Comment:
```suggestion
        public double Fuel
        {
            get
            {
                return (Body.FuelVolume);
            }
        }
```
и минус все проблемы

Reply:
не роботает